src/test/java/com/bitmovin/api/examples/CreatePerTitleEncoding.java

```diff
@@ -0,0 +1,135 @@
+        httpsInput = bitmovinApi.input.https.create(httpsInput);
+
+        S3Output s3Output = new S3Output();
+        s3Output.setBucketName(S3_OUTPUT_BUCKET_NAME);
+        s3Output.setSecretKey(S3_OUTPUT_SECRET_KEY);
+        s3Output.setAccessKey(S3_OUTPUT_ACCESS_KEY);
+        s3Output = bitmovinApi.output.s3.create(s3Output);
+
+        H264VideoConfiguration videoConfiguration = new H264VideoConfiguration();
+        videoConfiguration.setProfile(ProfileH264.HIGH);
+        videoConfiguration = bitmovinApi.configuration.videoH264.create(videoConfiguration);
+
+        videoConfiguration = bitmovinApi.configuration.videoH264.create(videoConfiguration);
+
+        Stream videoStream = new Stream();
+        videoStream.setCodecConfigId(videoConfiguration.getId());
+        videoStream.setMode(StreamMode.PER_TITLE_TEMPLATE);
+
+        InputStream inputStream = new InputStream();
+        inputStream.setSelectionMode(StreamSelectionMode.AUTO);
+        inputStream.setPosition(0);
+        inputStream.setInputPath(INPUT_PATH);
+        inputStream.setInputId(httpsInput.getId());
+        videoStream.setInputStreams(Collections.singleton(
+                inputStream
+        ));
+
+        videoStream = bitmovinApi.encoding.stream.addStream(encoding, videoStream);
+
+        FMP4Muxing fmp4Muxing = new FMP4Muxing();
+        fmp4Muxing.setOutputs(
+                Collections.singletonList(
+                        new EncodingOutput(
+                                s3Output.getId(),
+                                OUTPUT_BASE_PATH + "/video/{bitrate}/"
+                        )
+                )
+        );
+        fmp4Muxing.setStreams(
+                Collections.singletonList(
+                        new MuxingStream(
+                                videoStream.getId()
+                        )
+                )
+        );
+        fmp4Muxing.setSegmentLength(4.0);
+        bitmovinApi.encoding.muxing.addFmp4MuxingToEncoding(encoding, fmp4Muxing);
+
+        H264PerTitleConfiguration perTitleH264Configuration = new H264PerTitleConfiguration();
+
+        AutoRepresentation autoRepresentation = new AutoRepresentation();
+        autoRepresentation.setAdoptConfigurationThreshold(0.3);
+        perTitleH264Configuration.setAutoRepresentations(autoRepresentation);
+
+        PerTitle perTitle = new PerTitle();
+        perTitle.setH264Configuration(perTitleH264Configuration);
+
+        StartEncodingRequest startEncodingRequest = new StartEncodingRequest();
+        startEncodingRequest.setPerTitle(perTitle);
+
+        bitmovinApi.encoding.start(encoding, startEncodingRequest);
+
+        Task status;
+        do {
+            Thread.sleep(2500);
+            status = bitmovinApi.encoding.getStatus(encoding);
+        } while (status.getStatus() != Status.FINISHED && status.getStatus() != Status.ERROR);
+
+
+        List<Stream> perTitleResultStreams = bitmovinApi.encoding.stream.getStreams(encoding);
```

Comment:
will always be greater than 0 because the per title templates are also retrieved